plugins/modules/openssh_cert.py

```diff
@@ -112,7 +112,8 @@
             - "The point in time the certificate is valid from. Time can be specified either as relative time or as absolute timestamp.
                Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY-MM-DD | YYYY-MM-DDTHH:MM:SS | YYYY-MM-DD HH:MM:SS | always)
                where timespec can be an integer + C([w | d | h | m | s]) (for example C(+32w1d2h)).
-               Note that if using relative time this module is NOT idempotent."
+               Note that if using relative time this module is NOT idempotent.
+               The value C(always) is supported from OpenSSH 7.7."
```

Comment:
Theoretically `1970-01-01 00:00:00` should work as a substitute in this case. Would you be able to verify that and document that as an alternative for `OpenSSH <7.7`?

Reply:
It looks like `1970-01-01T00:00:00` is converted by some code to `always` automatically, and it fails again.

From `-vvv` output:

Arg given to `openssh_cert` module:
```
valid_from: '1970-01-01T00:00:00'
```
Command invoked has it changed to `always`:
```
-V always:20230101000000
```

Reply:
`1970-01-01T00:00:01` does work.